custom_modules/MongoDbController/MongoController.js

```diff
@@ -95,17 +95,8 @@ class MongoController
                         "occurred while parsing the results."
                     );
 
-                    let errMsg = "An unknown error occurred in " + 
-                                 "getMostRecent()";
-
-                    if (this.constructor && this.constructor.name)
-                    {
-                        errMsg = "An unknown error occurred in " + 
-                                 this.constructor.name + 
-                                 ".getMostRecent()";
-                    }
-
-                    reject(errMsg);
+                    reject(`An unknown error occurred in ` + 
```

Comment:
For better error messages (this just outputted "Function" rather than the MongoController's subclass name